index.html

```diff
@@ -108,7 +108,8 @@ <h1>What's in Standard?</h1>
             <ul class="list-group list-group-flush text-dark">
               <li class="list-group-item" v-bind:class="{ 'text-black-50': !isReleased(set) }" v-for="set in sets">
                 <div>
-                  {{ set.name }}
+                  <a v-if="set.code !== null" :href="'https://www.scryfall.com/sets/' + set.code.toLowerCase() + '?utm_source=whatsinstandard'">{{ set.name }}</a>
```

Comment:
Note that if a set has an assigned set code but no spoilers, users will get an error page. (Example: https://scryfall.com/sets/rna)

Reply:
Thanks for pointing this out. The benefit is nice so I'm fine trying this out, and we'll see if we get any bug reports.